.github/workflows/ci.yml

```diff
@@ -141,11 +141,24 @@ jobs:
       actions: 'read'
     steps:
     - name: Slack Workflow Notification
-      uses: Gamesight/slack-workflow-status@master
+      uses: slackapi/slack-github-action@v1.24.0
       with:
-        repo_token: ${{secrets.GITHUB_TOKEN}}
-        slack_webhook_url: ${{secrets.TBOX_SLACK_WEBHOOK}}
-        include_jobs: 'on-failure'  # only include jobs specifics if failure
+        payload: |
+          {
+              "text": "Scheduled GitHub CI result", # text which appears in desktop Slack message alert
+              "blocks": [
+                  {
+                      "type": "section",
+                      "text": {
+                          "type": "mrkdwn",
+                          "text": "${{ contains(needs.*.result, 'failure')  && '\u274C' || '\u2705' }} Scheduled GitHub CI: ${{ contains(needs.*.result, 'failure')  && '*Failure*' || '*Success*!' }}\nhttps://github.com/${{github.repository}}/actions/runs/${{github.run_id}}"
```

Comment:
Am I understanding this correctly that at present, this will post updates for both successful and failed runs? The goal will be to eventually make it just failed runs after testing?

Reply:
It will post both currently. If successes should not be posted, I could change that now (I can test on my fork now)

Reply:
Will now post to Slack only on failure.